src/main/java/com/mongodb/kafka/connect/util/ConnectionValidator.java

```diff
@@ -120,8 +120,15 @@ public void clusterDescriptionChanged(
       MongoClient mongoClient = MongoClients.create(mongoClientSettings);
 
       try {
```

Comment:
It would be possible to skip the `mongoClient.close()` calls by making this a try-with-resources instead:

```
try(MongoClient mongoClient = MongoClients.create(mongoClientSettings)) {
```